src/main/java/com/knighttodo/knighttodo/rest/TodoBlockResource.java

```diff
@@ -34,53 +34,53 @@
 public class TodoBlockResource {
 
     private final TodoBlockService todoBlockService;
-    private final TodoBlockMapper todoBlockMapper;
+    private final TodoBlockRestMapper todoBlockRestMapper;
 
-    @GetMapping("/block")
-    public ResponseEntity<List<TodoBlockResponse>> findAll() {
+    @GetMapping
+    public ResponseEntity<List<TodoBlockResponseDto>> getAllTodoBlocks() {
         log.info("Rest request to get all todo blocks");
 
         return ResponseEntity.status(HttpStatus.FOUND)
             .body(todoBlockService.findAll()
                       .stream()
-                      .map(todoBlockMapper::toTodoBlockResponse)
+                      .map(todoBlockRestMapper::toTodoBlockResponseDto)
                       .collect(Collectors.toList()));
     }
 
-    @PostMapping("/block")
-    public ResponseEntity<CreateTodoBlockResponse> addTodoBlock(
-        @Valid @RequestBody CreateTodoBlockRequest createRequest) {
-        log.info("Rest request to add todoBlock : {}", createRequest);
-        TodoBlockVO todoBlockVO = todoBlockMapper.toTodoBlockVO(createRequest);
+    @PostMapping
+    public ResponseEntity<CreateTodoBlockResponseDto> addTodoBlock(
+        @Valid @RequestBody CreateTodoBlockRequestDto createRequestDto) {
+        log.info("Rest request to add todoBlock : {}", createRequestDto);
+        TodoBlockVO todoBlockVO = todoBlockRestMapper.toTodoBlockVO(createRequestDto);
         TodoBlockVO savedTodoBlockVO = todoBlockService.save(todoBlockVO);
 
         return ResponseEntity.status(HttpStatus.CREATED)
-            .body(todoBlockMapper.toCreateTodoBlockResponse(savedTodoBlockVO));
+            .body(todoBlockRestMapper.toCreateTodoBlockResponseDto(savedTodoBlockVO));
     }
 
-    @GetMapping("/block/{todoBlockId}")
-    public ResponseEntity<TodoBlockResponse> getTodoBlockById(@PathVariable @Min(1) long todoBlockId) {
-        log.info("Rest request to get todoBlock by id : {}", todoBlockId);
-        TodoBlockVO todoBlockVO = todoBlockService.findById(todoBlockId);
+    @GetMapping("/{id}")
+    public ResponseEntity<TodoBlockResponseDto> getTodoBlockById(@Valid GetTodoBlockRequestDto getRequestDto) {
```

Comment:
You send Id but don't use it here. Use @PathVariable here and don't send a body to the GET method.

Reply:
so then we would not validate this path variable in controller and i can also remove this GetTodoBlockRequestDto class?